pallets/vector/src/tests.rs

```diff
@@ -21,13 +21,21 @@ use frame_support::{
 use frame_system::RawOrigin;
 use hex_literal::hex;
 use primitive_types::U256;
+use sp1_sdk::SP1ProofWithPublicValues;
 use sp_core::{crypto::AccountId32, keccak_256, ByteArray};
 use sp_runtime::{testing::H256, traits::BadOrigin};
 
 const TEST_SENDER_VEC: [u8; 32] =
 	hex!("d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d");
 const TEST_SENDER_ACCOUNT: AccountId32 = AccountId32::new(TEST_SENDER_VEC);
 
+// Generated with SP1 Helios https://github.com/succinctlabs/sp1-helios/blob/main/README.md
+// cargo prove key —-elf (sp1 helios elf) in SP1 Helios
+const SP1_VERIFICATION_KEY: [u8; 32] =
+	hex!("00788ce8dc2970920a3d3c072c8c07843d15f1307a53b3dd31b113c3e71c28e8");
```

Comment:
Note: I would consolidate the vkey into the `lib.rs` file so that it's not replicated in several places.